internal/mode/static/nginx/runtime/manager.go

```diff
@@ -17,18 +17,49 @@ import (
 )
 
 const (
-	pidFile            = "/var/run/nginx/nginx.pid"
+	PidFile            = "/var/run/nginx/nginx.pid"
 	pidFileTimeout     = 10000 * time.Millisecond
-	nginxReloadTimeout = 60000 * time.Millisecond
+	NginxReloadTimeout = 60000 * time.Millisecond
 )
 
 type (
-	readFileFunc  func(string) ([]byte, error)
-	checkFileFunc func(string) (fs.FileInfo, error)
+	ReadFileFunc  func(string) ([]byte, error)
+	CheckFileFunc func(string) (fs.FileInfo, error)
 )
 
 var childProcPathFmt = "/proc/%[1]v/task/%[1]v/children"
 
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . nginxPlusClient
```

Comment:
https://github.com/nginxinc/nginx-gateway-fabric/pull/1990 was just merged that changes how we specify these generation statements now, take a look.